src/utils.ts

```diff
@@ -1,10 +1,21 @@
 import deepmerge from 'deepmerge'
+import { toRaw } from 'vue'
 
 export type MergeOptionType = 'replaceArrays' | 'concatArrays'
 
+function deepToRaw(value: any): any {
+  if (Array.isArray(value)) {
+    return toRaw(value).map(deepToRaw);
+  }
+  if (typeof value === 'object' && value !== null) {
+    return Object.fromEntries(Object.entries(value).map(([key, value]) => ([key, deepToRaw(value)])))
+  }
+  return value;
```

Comment:
I don't think it's necessary, but I'm curious if the base case needs to call `toRaw()`. I can think of a few things that can't be handled, but I'm not sure we allow them anyways: Sets, Maps, etc.

Again, I think we're ok, just my brain went "If this is a deep `toRaw`, maybe we should call `toRaw` in our base case"